style.css

```diff
@@ -414,3 +415,209 @@
     background-color: #6070ff;
   }
 }
+
+@media screen and (min-width: 768px) {
+  #header {
+    height: 90px;
+    width: 100%;
+    padding-bottom: 40px;
+  }
+
+  .logo {
+    padding-left: 155px;
+    padding-top: 10px;
+  }
+
+  .nav {
+    --i1: scale(0);
+    --i2: scale(0);
+  }
+
+  .nav:has(.side-bar:target) {
+    --i1: scale(0);
+    --i2: scale(0);
+    --clip: inset(0 0 0 0);
+  }
+
+  .nav-container {
+    width: 100%;
+    margin: 0 auto;
+    display: flex;
+  }
+
+  .nav-menu {
+    display: none;
+  }
+
+  .nav-menu-second {
+    display: none;
+  }
+
+  .side-bar {
+    position: relative;
+    width: 100%;
+    background-color: #fff;
+    clip-path: none;
+    transition: none;
+    display: flex;
+    margin-right: 155px;
+  }
+
+  .side-bar li {
+    padding-right: 32px;
+    display: flex;
+    padding-bottom: 10px;
+  }
+
+  .side-bar li a {
+    color: rgba(52, 69, 99, 1);
+    font-size: 15px;
+  }
+
+  .social-container li a {
+    font-size: 16px;
+  }
+
+  .side-bar li a:hover {
+    color: rgba(52, 69, 99, 0.638);
+  }
+
+  .section-p1 {
+    background-image: url('img/Header-bg1.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top center;
+    height: 590px;
+    border-bottom: 1px solid rgba(223, 225, 230, 1);
+  }
+
+  .section-p1 .div1 {
+    margin-top: 90px;
+    margin-left: 360px;
+  }
+
+  .div1 h2 {
+    font-size: 48px;
+    width: 30%;
+    line-height: 60px;
+  }
+
+  .div1 p {
+    font-size: 20px;
+    width: 50%;
+  }
+
+  .section-p1 .div2 {
+    margin-left: 90px;
+    margin-left: 360px;
+  }
+
+  .section-p2 {
+    width: 100%;
+  }
+
+  .card {
+    width: 1156px;
+    height: 496px;
+    display: flex;
+    flex-direction: row;
+    border: 1px solid rgba(223, 225, 230, 1);
+  }
```

Comment:
- [ ] Kindly note the color and the gap between the project title, information, details and technologies in the Figma design and the gap between the project image and description.
- [ ] Make sure the project description content is aligned to the left 
- [ ] Please stick to the Figma design as much as possible as requested in the project requirement.

### Your design :point_down: 
![Screenshot from 2023-05-04 10-07-11](https://user-images.githubusercontent.com/91142432/236185258-c86767a7-1e3a-48de-a6ae-110a3d7623bc.png)
### Figma design :point_down: 
![Screenshot from 2023-04-06 10-25-14](https://user-images.githubusercontent.com/91142432/236185461-56b0ef37-993e-4292-861d-cafaec7ef499.png)